src/utils/github.ts

```diff
@@ -94,15 +94,21 @@ export class GithubApi {
   /**
    * Get content from the github repository
    */
-  async getContent(path: string): Promise<string> {
+  async getContent(path: string): Promise<string | null> {
     logger.info({ path }, 'GitHub API: Get Content');
-    const response = await this.octokit.rest.repos.getContent({ owner: this.owner, repo: this.repo, path });
-    if (!this.isOk(response.status)) throw new Error('Failed to get aerial TileSet config.');
-    if ('content' in response.data) {
-      return Buffer.from(response.data.content, 'base64').toString();
-    } else {
-      throw new Error(`Unable to find the content from path ${path}.`);
+    try {
+      const response = await this.octokit.rest.repos.getContent({ owner: this.owner, repo: this.repo, path });
+      if (this.isOk(response.status)) {
+        if ('content' in response.data) {
+          return Buffer.from(response.data.content, 'base64').toString();
+        } else {
+          throw new Error(`Unable to find the content from path ${path}.`);
+        }
+      }
+    } catch {
+      logger.debug({ path }, 'GitHub: Content not found');
```

Comment:
Why throw an exception and catch it again immediately?

Reply:
We need the above error if the content not exist in data.

Reply:
I mean, why not log it immediately, rather than throwing and catching?

Reply:
Good point, just added that.

Reply:
I think this should be error if response content not exists. If we log and return null. This error won't been capture.